.gitattributes

```diff
@@ -8,3 +8,4 @@
 .gitattributes export-ignore
 .gitpod.yml export-ignore
 CHANGELOG.md export-ignore
+/package.json export-ignore
```

Comment:
@bennothommo thoughts?

The rational being with the new `*:config` command, the mix/vite deps can be added dynamically and therefore we don't need to ship a default package.json

Reply:
@jaxwilko yeah makes sense to do this. We'll just need to perhaps document this for people who wish to develop Winter CMS, that they will need to run the `*:config` commands if they wish to develop anything built by Vite/Mix.